openbas-api/src/main/java/io/openbas/migration/V3_41__Add_column_expiration_time_expectations.java

```diff
@@ -0,0 +1,34 @@
+package io.openbas.migration;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+import java.sql.Connection;
+import java.sql.Statement;
+
+@Component
+public class V3_41__Add_column_expiration_time_expectations extends BaseJavaMigration {
+
+  @Override
+  public void migrate(Context context) throws Exception {
+    Statement select = context.getConnection().createStatement();
+    Connection connection = context.getConnection();
+    Statement statement = connection.createStatement();
+    long technicalMinutesExpirationTime = 60L;
+    long manualMinutesExpirationTime = 360L;
```

Comment:
This is in minute or second ?
To compare with what we have as default in Contract

Reply:
It is in minutes, as mentionned in the name.